docs/user/dql/functions.rst

```diff
@@ -1597,21 +1597,21 @@ Description
 >>>>>>>>>>>
 
 Usage: hour(time) extracts the hour value for time. Different from the time of day value, the time value has a large range and can be greater than 23, so the return value of hour(time) can be also greater than 23.
+The function `hour_of_day` is also provided as an alias.
```

Comment:
do we need an underscore to use it as a link?

Reply:
Yes we do, but it does not have its own entry in the doc so I don't believe it would be linking to anything. Should I just remove the backticks altogether?